storage/innobase/xtrabackup/test/t/xb_duplicate_sdi.sh

```diff
@@ -40,5 +45,18 @@ mysql -e "ALTER TABLE t1 IMPORT TABLESPACE;" test
 
 backup_and_restore test
 
+vlog "case#3 check prepare works with backup of 8.0.23"
+stop_server
+#cleanup backup and place 8.0.23 backupdir
+rm -rf $topdir/backup && mkdir -p $topdir/backup
+run_cmd tar -xf inc/duplicate_sdi_backup.tar.gz -C $topdir/backup
+#prepare
+run_cmd xtrabackup --prepare --target-dir=$topdir/backup
+#restore
+[[ -d $mysql_datadir ]] && rm -rf $mysql_datadir && mkdir -p $mysql_datadir
+xtrabackup --copy-back --target-dir=$topdir/backup
+
+start_server
```

Comment:
can you please check the contents of table here (after starting server)? or table checksums?

Reply:
done.